runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant.service.ts

```diff
@@ -75,6 +75,20 @@ export class TenantService {
         return this.tenantRepository.save(updatedTenant);
     }
 
+    async validateInviteCode(inviteCodeDto: TenantInviteCodeDto) {
+        const inviteCode = await this.inviteCodeRepository.findOneOrFail({
+            where: {
+                inviteId: inviteCodeDto.inviteCode,
+                expirationDate: MoreThanOrEqual(new Date())
+            },
+            relations: ['tenant']
+        }).catch(() => {
+            throw new BadRequestException('Invite code not valid or expired');
+        });
+
+        return { tenantName: inviteCode.tenant.name };
```

Comment:
why do you return tenantName instead of boolean (thats what the validateInviteCode name suggests)